test/unit/rules/sort-invocations-test.js

```diff
@@ -1746,6 +1746,152 @@ generateRuleTests({
         `);
       },
     },
```

Comment:
These should all be different tests to help with the debug output size when things go wrong

Reply:
The formatting in this file is due to two things:

- I used the same groupings that I had used in [`ember-codemod-sort-invocations`](https://github.com/ijlee2/ember-codemod-sort-invocations/blob/1.0.5/tests/fixtures/my-app/input/app/components/example-15.hbs).
- `ember-template-lint` project doesn't have a history of separating a large file into multiple (e.g. utilities, test files) to help with readability, so I didn't want to introduce a new pattern in my pull request.